src/ui/components/App.vue

```diff
@@ -50,7 +50,7 @@
       <v-btn
         text
         small
-        @click="() => openExternal('https://github.com/baruchiro/israeli-bank-scrapers-desktop')"
+        @click="() => openExternal({appRepository})"
```

Comment:
```suggestion
        @click="() => openExternal(appRepository)"
```

The `openExternal` is just a function in another place, and it gets a URL as arguemnt, so you don't need the `{ }` (if you use them, it actualy mean that you passed an object with field `appRepository`, instead of only string)